scripts/install-deps.sh

```diff
@@ -852,6 +854,13 @@ while [[ $1 != "" ]]; do
     shift
 done
 
+# setup workfolder if different from default
```

Comment:
im not sure i understand why this is needed. shouldn't a mkdir provide appropriate rights?